keyboards/keebio/bfo9000/keymaps/deepbluealchemy/keymap.c

```diff
@@ -0,0 +1,250 @@
+#include QMK_KEYBOARD_H
+#include "split_util.h"
+
+extern keymap_config_t keymap_config;
+
+enum bfo9000_layers {
+  _QWERTY,    // Qwerty
+  _GIT,       // Git
+  _GAME       // Game
+};
+
+enum bfo9000_keycodes {
+  QWERTY = SAFE_RANGE,
```

Comment:
Since you only have the one base layer, you can remove this, without any issues.